libraries/botbuilder-core/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@botbuildercommunity/core",
+  "version": "1.0.0",
+  "description": "Classes used by other (community) libraries.",
+  "main": "./lib/index.js",
+  "typings": "./lib/index.d.ts",
+  "directories": {
+    "lib": "lib"
+  },
+  "nyc": {
+    "extends": "@istanbuljs/nyc-config-typescript",
+    "all": true,
+    "check-coverage": true,
+    "lines": 0,
+    "statements": 0,
+    "functions": 0,
+    "branches": 0,
+    "exclude": [
+      "**/*.d.ts",
+      "**/*.test.js"
+    ],
+    "reporter": [
+      "text",
+      "cobertura",
+      "html"
+    ],
+    "report-dir": "./.nyc_output/coverage"
+  },
+  "scripts": {
+    "build": "tsc",
+    "test": "tsc && nyc mocha",
+    "test-ci": "tsc && nyc mocha --reporter xunit --reporter-options output=./mocha-test-results.xml --reporter-options suiteName=$npm_package_name"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/BotBuilderCommunity/botbuilder-community-js.git"
+  },
+  "contributors": [
+    "Bot Builder Community",
+    "Mick Vleeshouwer <mick@imick.nl>",
+    "Néstor Alonso Viña <nalonso@student.42.fr>"
```

Comment:
@nestoralonsovina is this ok for you? I added you a while ago as a contributor to this project, however I have never verified if you would like to have your full name / this email address in here. 

Could you please verify and/or let me know how you would like to be credited.

Reply:
Sure, that looks good to me! Thanks